modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+  direction: Ingress                                <4>
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType] <5>
+  filters:                                          <6>
+  - field: SrcSubnetLabel
+    matchType: Absence
+----
+<1> The same namespace, `netobserv`, is used across all example components in this documentation. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+<2> How the metrics appear, along with the prefix `netobserv`. 
+<3> The `type` `Counter` is useful for counting bytes or packets. Histogram is the other available type, which is useful for latencies. 
+<4> Direction of traffic to capture. Egress is an alternative option. If not specified, both ingress and egress are captured, which can lead to duplicated counts. 
+<5> Labels define what the metrics look like and the relationship between the different entities. The labels also define the metrics cardinality. For example, `SrcK8S_Name` is a high cardinality metric.
+<6> Filters allow further refinement of what you want to see. In this example,  selecting only the cluster external traffic is done by matching only flows where `SrcSubnetLabel` is absent. This assumes the subnet labels feature is enabled (via `spec.processor.subnetLabels`), which is done by default.
+
+.Verification
+. Once the pods refresh, Navigate to *Observe* -> *Metrics*.
+. In the Expression field, you can type the metric name to see it displayed. 
+. You can also enter an expression, such as `topk(5, sum(rate(netobserv_cluster_external_ingress_bytes_total{DstK8S_Namespace="my-namespace"}[2m])) by (DstK8S_HostName, DstK8S_OwnerName, DstK8S_OwnerType))`
+====
++
+.Show RTT latency for cluster external ingress traffic
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-rtt
+  namespace: netobserv    <1>
+spec:
+  metricName: cluster_external_ingress_rtt_seconds
+  type: Histogram                 <2>
+  valueField: TimeFlowRttNs
+  direction: Ingress
+  labels: [DstK8S_HostName,DstK8S_Namespace,DstK8S_OwnerName,DstK8S_OwnerType]
+  filters:
+  - field: SrcSubnetLabel
+    matchType: Absence
+  - field: TimeFlowRttNs
+    matchType: Presence
+  divider: "1000000000"      <3>
+  buckets: [".001", ".005", ".01", ".02", ".03", ".04", ".05", ".075", ".1", ".25", "1"]  <4>
+----
+<1> The same namespace, `netobserv`, is used across all example components in this documentation. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+<2> The `type` here is `Histogram`, since it looks for a latency value (`TimeFlowRttNs`).
+<3> Since the Rount-trip time (RTT) is provided as nanos in flows, use a divider of 1 billion to convert into seconds, which is standard in Prometheus guidelines.
```

Comment:
I am guessing it's `round`.
```suggestion
<3> Because the round-trip time (RTT) is provided as nanos in flows, specify a divider of 1 billion to convert into seconds, which is standard in Prometheus guidelines.
```

Reply:
Lol yes, the "t" in trip got ahead of me.